kms/cloudkms/attestation.go

```diff
@@ -0,0 +1,641 @@
+package cloudkms
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"crypto"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/binary"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"math"
+	"regexp"
+	"slices"
+	"strconv"
+	"time"
+
+	"cloud.google.com/go/kms/apiv1/kmspb"
+
+	"github.com/pkg/errors"
```

Comment:
This occurrence of `pkg/errors` doesn't seem to be required; there's just `errors.New` calls.

Reply:
Fixed with 26a643e